app/views/autosign/_counts.html.erb

```diff
@@ -0,0 +1,22 @@
+<div class="col-md-3">
+  <div class="card-pf  card-pf-accented card-pf-aggregate-status card-pf-with-action">
+    <h2 class="card-pf-title">
+      <a href="#autosign">
+        <%= icon_text('check', '', :kind => 'fa') %><span class="card-pf-aggregate-status-count"><%= autosign.length.to_s %></span> <%= _("Autosign entries") %>
```

Comment:
this card is only displayed for users with the `:view_smart_proxies_autosign` permission